x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/transport/SecurityServerTransportInterceptorTests.java

```diff
@@ -713,7 +713,7 @@ public void testSendWithUserIfRemoteAccessHeadersConditionNotMet() throws Except
         final AuthenticationTestHelper.AuthenticationTestBuilder builder = AuthenticationTestHelper.builder();
         final Authentication authentication;
         if (unsupportedAuthentication) {
-            authentication = builder.serviceAccount().build();
+            authentication = builder.remoteAccess().build();
         } else {
             authentication = randomFrom(
                 builder.apiKey().build(),
```

Comment:
Should we include `builder.serviceAccount().build()` in the supported list?

Reply:
Yes good point!